laravel/app/Order.php

```diff
@@ -10,30 +10,25 @@ class Order extends Model
     protected $primaryKey = 'id';
     public $timestamps = false;
 
-    public function scopeOrderIndex($query, $item_id, $delivery_user_id, $order_start, $order_end)
+    public function orderItem()
```

Comment:
hasManyであれば復数系にしましょう
```suggestion
    public function orderItems()
```